dags/events_to_amplitude.py

```diff
@@ -51,9 +49,9 @@ def key_file(project):
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/events_to_amplitude.sh",
     dag=dag)
 
-devtools_events_to_amplitude = EMRSparkOperator(
-    task_id="devtools_events_to_amplitude",
-    job_name="DevTools Events to Amplitude",
+devtools_prerelease_events_to_amplitude = EMRSparkOperator(
+    task_id="devtools_prerelease_events_to_amplitude",
+    job_name="DevTools Prerelease Events to Amplitude",
     execution_timeout=timedelta(hours=8),
     instance_count=DEVTOOLS_INSTANCES,
```

Comment:
nit: `PRERELEASE_DEVTOOLS_INSTANCES`?

Reply:
I was going to use the same number I can get the job to run fast enough on release :D I'll change it in the follow-up PR if that's not the case